prestep.F90

```diff
@@ -58,15 +58,13 @@ subroutine prestep(carma, cstate, rc)
           ! (i.e. supersaturated) and d_gc is negative. For better performance,
           ! substep the gas unless both of these are true. This might run into
           ! trouble if d_t is large and negative.
-!          if (d_gc(iz, igas) < 0._f) then
-          if ((d_gc(iz, igas) < 0._f) .and. ((supsatiold(iz, igas) > 0._f) &
-               .or. (supsati(iz, igas) > 0._f))) then
```

Comment:
and this line

Reply:
Wasn't the change in the other direction (i.e. to simplify these and ignore the supersat limits)? I am a little confused with what we started with and what we changed here.

Reply:
is done